README.md

```diff
@@ -2,11 +2,12 @@
 
```

Comment:
- [ ] OPTIONAL
The details of **Authors** under the _Author_ section of your ReadMe is not properly arranged. To make your ReadMe more cleaner and kindly endeavor to add the **full names** of the authors and each detail of an author should appear on a single line as can be seen in [this template](https://github.com/microverseinc/readme-template).

Reply:
This issue is well addressed ✔️